test/e2e/framework/ginkgowrapper.go

```diff
@@ -235,24 +275,43 @@ func registerInSuite(ginkgoCall func(string, ...interface{}) bool, args []interf
 			ginkgoArgs = append(ginkgoArgs, arg)
 		}
 	}
-	offset += 2 // This function and its direct caller.
 
-	// Now that we have the final offset, we can record bugs.
+	// For top-level DSL functions. Everything beneath those uses GinkgoHelper.
+	offset := 1 // TODO: does not quite work yet?!
```

Comment:
@onsi: how does stack unwinding handle a mixture of offset and GinkgoHelper?

This code should be called via `ginkgo.It (no helper because I didn't want to modify all of them) -> transformNewNodeArgs (helper) -> transformGinkgoNodeArgs (my code, helper) -> expandGinkgoArgs (helper) -> NewBug (no helper, adds one to offset) -> NewCodeLocation`.

Does `NewCodeLocation` decrement the offset that it is given for each parent which is not a helper (NewBug, It) or does first use the offset, then the helper flag?

I seem to get the code location of `ginkgo.It` no matter which offset value I use here, so it seems to be the latter behavior.